chia-protocol/src/to_json_dict.rs

```diff
@@ -6,23 +6,28 @@ pub trait ToJsonDict {
     fn to_json_dict(&self, py: Python) -> PyResult<PyObject>;
 }
 
-impl ToJsonDict for bool {
-    fn to_json_dict(&self, py: Python) -> PyResult<PyObject> {
-        Ok(self.to_object(py))
-    }
-}
-
-impl ToJsonDict for u32 {
-    fn to_json_dict(&self, py: Python) -> PyResult<PyObject> {
-        Ok(self.to_object(py))
-    }
+macro_rules! from_json_primitive {
```

Comment:
```suggestion
macro_rules! to_json_primitive {
```

Did you intend to use this macro name?

Reply:
oops :)